mos-platform/geos-cbm/clang.cfg

```diff
@@ -0,0 +1,5 @@
+-mllvm -inline-threshold=0
+-T vlir.ld
+-mlto-zp=107
+-Wl,-linit-stack,--no-check-sections
```

Comment:
We've used a "trick" to avoid `--no-check-sections`, and I think it should probably work here too.

All llvm-mos LLD linker addresses are 32 bits, regardless of how the address is actually used. Relocations will intentionally silently discard the high two bytes. This allows the high two bytes to encode banking information, and it simplifies linker scripts by giving every section a distinct link-time address.

Conventionally, we've used the highest byte as a "category" or "namespace", and the low byte as a "bank number". In particular, it's fairly straightfoward to access the bank number of an address in assembly; we'd like to extend this to C too once we figure out how to spell it.

Reply:
Hi, i am not really sure that i got your point. As it is a little bit more work, let me ask first:

Are you saying i should start the overlay sections at 0x010000, 0x020000, 0x030000, .... ?

So this change in the code:

SECTIONS { 
  
  INCLUDE c.ld 

  .dir_block : { *(.dir_block .dir_block.*) } >dir_block
  .info_block : { *(.info_block .info_block.*) } >info_block

  .vlir01 __overlayaddr +0x010000 : AT (__overlayaddr+0*__overlaysize) { *(.vlir01 .vlir01.*) NOCROSSREFS(.vlir01 .vlir02 .vlir03 .vlir04 .vlir05 .vlir06 .vlir07 .vlir08 .vlir09 .vlir0a .vlir0b .vlir0c .vlir0d .vlir0e .vlir0f) } > vlir01
  .vlir02 __overlayaddr +0x020000 : AT (__overlayaddr+1*__overlaysize) { *(.vlir02 .vlir02.*) NOCROSSREFS(.vlir01 .vlir02 .vlir03 .vlir04 .vlir05 .vlir06 .vlir07 .vlir08 .vlir09 .vlir0a .vlir0b .vlir0c .vlir0d .vlir0e .vlir0f) } > vlir02
 
...

Reply:
Yep, that's correct.